lib/rspec/core/metadata.rb

```diff
@@ -47,7 +47,9 @@ def self.relative_path(line)
         return nil if line == '-e:1'.freeze
         line
       rescue SecurityError
+        # :nocov:
         nil
+        # :nocov:
       end
```

Comment:
Isn't `nil` the default return value from an empty `rescue` clause?  If that's correct, then we could just remove the `nil` line entirely as it doesn't serve a purpose.